full-v3.yml

```diff
@@ -0,0 +1,2299 @@
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolsResponse'
+
+  /schools/{id}:
+    get:
+      tags: ["Schools"]
+      description: Returns a specific school
+      operationId: getSchool
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /schools/{id}/courses:
+    get:
+      tags: ["Schools"]
+      description: Returns the courses for a school
+      operationId: getCoursesForSchool
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/CoursesResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /schools/{id}/district:
+    get:
+      tags: ["Schools"]
+      description: Returns the district for a school
+      operationId: getDistrictForSchool
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/DistrictResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /schools/{id}/sections:
+    get:
+      tags: ["Schools"]
+      description: Returns the sections for a school
+      operationId: getSectionsForSchool
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SectionsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /schools/{id}/terms:
+    get:
+      tags: ["Schools"]
+      description: Returns the terms for a school
+      operationId: getTermsForSchool
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/TermsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /schools/{id}/users:
```

Comment:
Based on Amelie's spec, this endpoint should support ?role=... right?
https://docs.google.com/document/d/1gYdhGs2LjgFWBVbgXTPa1VjOTK8qLvo1Rvfl_YozBww/edit?disco=AAAAHScd60o
Caveat: we currently haven't built /schools/{id}/users?role=contact support. The not DB-modifying solution is something we tabled, and I still think it's kind of bad (grab all students in school X, search all contacts where contact.students is in the massive array; the query is doable, but I feel like it will break?).